global.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*';
```

Comment:
Why this?

Reply:
This was due to a number of modules not having type definitions. It's a bit unsafe though, ideally we should declare each package that doesn't have type definitions (or maybe look to contribute back to them)